week-1-ts-ecosystem/w01e03-object-oriented-structure/src/test/cart.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { Cart, CartItem } from '../model/cart'
+
+describe("Cart", () => {
```

Comment:
Bardzo fajnie, że wprowadziłeś @Dam1anM testy.

To jest dobre zadanie, na sprawdzenie przez testowanie.

Jeśli chodzi o same testy to możesz też przygotować sobie ich tyle ile potrzeba w osobnych plikach np:
w zadaniu jest o 3 rodzajach koszyka to możesz rozdzielić do 3 plików:

`buy-now-cart.test.ts`
`auction-cart.test.ts`
`for-free.test.ts`

Reply:
@michaljabi a co byś powiedział jakby takie testy "per rodzaj koszyka" zamiast dzielić na pliki to podzielić je w pliku z testami używając bloków describe?

Reply:
Też będzie git. Pewnie nawet wygodniej bo będziesz mógł z tych samych danych korzystać :)

Reply:
Dodałem kilka testów i zrobiłem mały podział. Już nie pisałem osobnych scenariuszy na każdy koszyk